src/scripts/num.js

```diff
@@ -141,6 +141,15 @@ export function uatoms(number = 0) {
   )
 }
 
+export function toMicroUnit(number, viewDenom, network) {

```

Comment:
it feels wrong to me that we're dealing with number conversions in the FE. I'm not up to date on the state of the transaction service but it feels like the API should be handling this change because the user never needs to see microunits in the frontend.

Reply:
you need to sign client side. so visual representation needs to be converted into sth signable which requires a conversion.

we could prepare the sign message in the API but that could cause man in the middle attacks where someone serves a wrong message and let's the user sign it without the user knowing.

Reply:
Agree with this point!

Reply:
which point?